forms/src/pages/SurveyCreationPage/SurveyCreationPage.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import SurveySection from "./components/form/SurveySection/SurveySection";
+import SurveyTitle from "./components/form/SurveyTitle/SurveyTitle";
+import "./SurveyCreationPage.scss";
+import plus from "../../assets/plus.svg";
+import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
+import { iQuestion, QuestionType } from "../../interfaces/iQuestion";
+import { sectionsContext } from "../../context/sectionsContext";
+
+function SurveyCreationPage(props: { surveyName: string }) {
+  const mock: iQuestion = {
+    id: "123456123456123456123456",
+    questionName: "מה קורה?",
+    questionType: QuestionType.radio,
+    answers: [
+      { answer: "בסדר" },
+      { answer: "" },
+      { answer: "על הפנים" },
+      { answer: "שורד." },
+    ],
+  };
+
+  const [sections, setSections] = useState([mock]);
+
+  const addSection = () => {
+    setSections([
+      ...sections,
+      {
```

Comment:
This mock question is hardcoded several times despite being in your mock file. Export and import it instead.